doc/CDN_DEPLOYMENT.md

```
@@ -0,0 +1,248 @@
+| :------------- | :---------------------------------- | :--------------------------------- |
+| 静态资源缓存   | `URI Path matches /assets/*`        | Cache Everything, Edge TTL: 1 year |
+| JS/CSS 缓存    | `URI Path matches /*.js` 或 `*.css` | Cache Everything, Edge TTL: 1 year |
+| HTML 绕过      | `URI Path matches /index.html`      | Bypass Cache                       |
+| API 绕过       | `URI Path starts with /api/`        | Bypass Cache                       |
+| WebSocket 直通 | `URI Path starts with /ws/`         | Bypass Cache                       |
+
+### 3. Page Rules（可选）
+
+如果使用免费计划，可使用 Page Rules（最多 3 条免费）：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)